server/modules/auth/api/authApi.js

```diff
@@ -64,14 +47,6 @@ export const init = (app) => {
     }
   })
 
-  app.post('/auth/login', async (req, res, next) => {
```

Comment:
Why login is outside ( in arena-server) but /auth/reset-password not?

Reply:
all the other ones could be moved to arena-server too. I have to use the one in arena-server otherwise passport will be initialized 2 times!